flock test failed

**What happened**:
fslock test failed .
=== CONT  TestExec
    lock_test.go:778: lock_test.go:186: wrote initial output file to [/jfs/gofslock/fslock/multiprocessing252973008/out]
    lock_test.go:778: lock_test.go:2[66](https://github.com/juicedata/juicefs/runs/7835179646?check_suite_focus=true#step:9:67): subprocesses reported 1 failure(s)
--- FAIL: TestExec (6.87s)
**What you expected to happen**:
The case passed .
**How to reproduce it (as minimally and precisely as possible)**:
Follow the action https://github.com/juicedata/juicefs/runs/7835179646?check_suite_focus=true .
**Anything else we need to know?**
No .

--- a/pkg/utils/clock_test.go
+++ b/pkg/utils/clock_test.go
@@ -28,7 +28,7 @@ func TestClock(t *testing.T) {
 	}
 	c1 := Clock()
 	c2 := Clock()
-	if c2-c1 > time.Microsecond {
-		t.Fatalf("clock is not accurate: %s", c2-c1)
+	if c2 < c1 {
+		t.Fatalf("clock is not monotonic: %s > %s", c1, c2)
 	}
 }

